draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +446,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
 
 From a coherent pluggable data modeling perspective, a set of attributes is grouped together and represented by a single identifier known as the "Supported Mode." In essence, each supported mode encapsulates a combination of functional capabilities for coherent pluggables, such as modulation type, bit rate, baud rate, chromatic dispersion, polarization, FEC, and more. Some of these attributes may also include value ranges (e.g., minimum and maximum). A coherent pluggable can support multiple supported modes, each of which can be defined by one of the following methods:
```

Comment:
we did not define what we'd count as "functional capabilities" in general. I would assume these are related to pluggables in general and not limited to the media side only.